custom/icds_reports/dashboard_utils.py

```diff
@@ -60,3 +61,208 @@ def _get_nav_metadatada():
             'image': static('icds_reports/mobile/images/bulb.png')
         },
     }
+
+
+def _get_nav_menu_items():
+    return {
+        'Maternal and Child Nutrition': [
+            {
+                'name': 'Prevalence of Underweight (Weight-for-Age)',
+                'route': 'maternal_and_child/underweight_children',
+                'children': False,
```

Comment:
I don't see any instances of `'children'` being anything other than `False`. What's it for?

Reply:
Actually do any of these ever change? If not that's definitely a reason to use `attrs` or even just remove them and only add them back when needed

Reply:
> I don't see any instances of `'children'` being anything other than `False`. What's it for?

Sorry, forgot to remove it. No use for this.